enforcer-parent/enforcer/src/main/java/org/wso2/choreo/connect/enforcer/constants/APISecurityConstants.java

```diff
@@ -168,5 +168,8 @@ public static String getFailureMessageDetailDescription(int errorCode, String er
 
     public static final int DEFAULT_MAX_VALID_KEYS = 250;
     public static final int DEFAULT_MAX_INVALID_KEYS = 100;
+
+    public static final int API_AUTH_INVALID_AUDIENCE = 901104;
```

Comment:
shall we reuse the 901102 status code here as per discussion

Reply:
Done via https://github.com/wso2/product-microgateway/pull/3357/commits/55cc1cf5b40ccd3a3d161aa6ae790780a8d79264